crates/ra_ide/src/ssr.rs

```diff
@@ -27,11 +27,11 @@ impl std::error::Error for SsrError {}
 //
 // Search and replace with named wildcards that will match any expression.
 // The syntax for a structural search replace command is `<search_pattern> ==>> <replace_pattern>`.
-// A `$<name>:expr` placeholder in the search pattern will match any expression and `$<name>` will reference it in the replacement.
+// A `$<name>` placeholder in the search pattern will match any AST node and `$<name>` will reference it in the replacement.
 // Available via the command `rust-analyzer.ssr`.
 //
 // ```rust
-// // Using structural search replace command [foo($a:expr, $b:expr) ==>> ($a).foo($b)]
+// // Using structural search replace command [foo($, $b) ==>> ($a).foo($b)]
```

Comment:
*$a

Reply:
Thanks. I amended the commit and force pushed. Hopefully that was the right thing to do. I've not done a lot of PRs on github, so not really sure of etiquette.

Reply:
We generally are pretty lax as far as workflow and code style goes, "best practicies" are being documented here:

https://github.com/rust-analyzer/rust-analyzer/blob/master/docs/dev/README.md#code-style--review-process